### PR TITLE
Index dot files by default but expose option to config.

### PR DESCRIPTION
**Pull request by kellegous**

### CODE REVIEW

File: config-example.json [hunk 1]
```diff
@@ -6,7 +6,8 @@
         },
         "AnotherGitRepo" : {
             "url" : "https://www.github.com/YourOrganization/RepoOne.git",
-            "ms-between-poll": 10000
+            "ms-between-poll": 10000,
+            "excluded-dot-files": true
```
Comment (jklein) [inline config-example.json]: Should this be "exclude-dot-files" to match the Go and be a little more intuitive?

File: config/config.go [hunk 2]
```diff
@@ -25,6 +25,7 @@ type Repo struct {
 	Vcs              string         `json:"vcs"`
 	VcsConfigMessage *SecretMessage `json:"vcs-config"`
 	UrlPattern       *UrlPattern    `json:"url-pattern"`
+	ExcludeDotFiles  bool           `json:"exclude-dot-files"`
```
Comment (jklein) [inline config/config.go]: Yeah this makes me think the above is just a typo.

File: config-example.json (hunk 1 above)
Comment (kellegous) [inline config-example.json]: Thanks, that's definitely a typo.

**Comment by jklein** [comment]
LGTM

**Pull request closed without merging**